source/Calamari.Aws/Calamari.Aws.csproj

```diff
@@ -27,6 +27,7 @@
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Autofac" Version="4.8.0" />
```

Comment:
I would prefer to keep the `Autofac` reference out of the extensions/plugins. If we don't and we want to update it, it'll e a pain. I think it would be enough that the above class returned a list of types to register `.AsSingleInstance().AsImplementedInterfaces().AsSelf()`